temporalcli/commands.taskqueue.go

```diff
@@ -10,7 +10,7 @@ import (
 	"go.temporal.io/api/enums/v1"
 	"go.temporal.io/api/taskqueue/v1"
 	"go.temporal.io/api/workflowservice/v1"
-	"go.temporal.io/server/common/tqname"
+	"go.temporal.io/server/common/tqid"
```

Comment:
IMO we should open a ticket to stop referencing server code outside of devserver (if possible).